### ISSUE TITLE
PaleMoon+some bugs

### ISSUE THREAD

**Issue by Tnebi** [uBlock-LLC/uBlock#770]
Hello I'm using PaleMoon as default browser and as for ABP and ABE it doesn't show the icon. Because seems that since those icons are not 'static' they're not showing into PaleMoon. Wondering if you can take a look and make a version for it. Also I've noticed in my firefox that the 'right-click option' to block elements isn't really working well and there's no comeback from that option(so you have to reload the page)

**Comment by gorhill** [comment]
Dup of #518:

> As I said at the SeaMonkey request, the only problem is the toolbar button and the popup.

**Comment by tailHey** [comment]
Hello everyone.
Sorry, but I'm afraid this isn't just one problem to notice.
It seems that cosmetic filtering doesn't work properly:

Go to http://raymondhill.net/ublock/tests.html , select Network + cosmetic filtering, below the "There should be no annoying whole page shifting upward (specific cosmetic filtering)", an ad can be seen (whereas it's blocked on Opera Next 28).

Maybe it's not completely related to cosmetic filtering, here is another example: Go to http://twog.fr/le-comptwoir-de-nikal83/ , you'll notice that Twitter widgets aren't blocked, whereas they're blocked on Opera Next 28.

Pale Moon 25.2.1 x64, µBlock 0.8.7.0, custom settings.
Opera Next 28, µBlock 0.8.7.0, same custom settings + µMatrix (custom settings so that it can works fine with µBlock).
Both tested on the same computer, Windows 8.1 x64 up to date.
Let me know if you need more data.

**Comment by JeremyWW** [comment]
As per #518 be good to have this done.

**Comment by JeremyWW** [comment]
Not sure why you guys are using the alternative browsers - for me it's because Pale Moon is native 64 bit. Just came across 'Waterfox' - https://www.waterfoxproject.org/ which is another, seemingly fast, 64 bit Firefox build only this uses the current FF UI so uBlock appears to be fully compatible.

**Comment by littleboy62** [comment]
Hi everyone. ;)

@JeremyWW

> Not sure why you guys are using the alternative browsers

So, if I understand, you must use only **IE.**
It's a **joke**, _don't hurt me_ :D

You talk about _waterfox_, a another alternative Browser.
**I'll test after**.

> for me it's because Pale Moon is native 64 bit.

For me, it's _an another reason_.
Because there is **an AVX build available for Pale Moon**, who works great.

This browser (AVX Build) is **more faster,  stable** than _web browsers  I've tested_.

So thanks, i'll soon download Waterfox.

Best regards,
Thomas.

**Comment by Tnebi** [comment]
Waterfox is just Firefox 64 bit while PaleMoon is a bit different. It has his own UI, optimizations and it is really stable and faster for me.

**Comment by tailHey** [comment]
Update about my post ( https://github.com/gorhill/uBlock/issues/770#issuecomment-74056281 ):

My bad, I disabled µMatrix on my Opera Next, and I got the exact same results (for both the things I mentioned).
So I don't know if this is suppose to work like this, that's to say to see some web content below the "There should be no annoying whole page shifting upward (specific cosmetic filtering)" label and to see that Twitter widgets aren't blocked...

**Comment by gorhill** [comment]
I closed this bug as dup of #518.

Why are supposed problems about twitter widgets reported here?

**Comment by JeremyWW** [comment]
@gorhill Who knows... :-/

**Comment by JeremyWW** [comment]
@Tnebi Pale Moon's UI is just the 'old' Firefox UI which gives Waterfox the thumbs up for me especially as uBlock works properly (or seems to...). Also, Flash loads faster in Waterfox - I'm not getting any Flash 'hangups' as I did with FF and PM. Anyway, this is really getting off topic and this issue is closed anyway.

**Comment by tailHey** [comment]
@gorhill: Well, I considered that Twitter widgets are as privacy-risky as Facebook ones so since I checked a lot of filters (anti-tracking ones included) I thought it was wrong to let µBlock allow them; was I wrong?

**Comment by gorhill** [comment]
@tailHey This bug is about Palemoon not showing uBlock in the toolbar, nothing to do with Twitter widgets. Anything unrelated to OP's bug need to go in a new issue.

And in any case I didn't nee anything wrong, the "widgets" on that specific site to me seem to have specific classes/ids which are not found in the cosmetic filters of Fanboy's Social.

And as far as privacy go, Fanboy's Social is certainly not fit for that purpose: cosmetic filtering has no value privacy-wise. The only sure way to take privacy into your own hands is dynamic filtering, it will block with 100% certainty connections to `twitter.com`, `facebook.com`, and whatever domain is deemed a privacy risk.

**Comment by tailHey** [comment]
@gorhill: Sorry about being off-topic here.

Yeah I know what you mean. That's why I also µMatrix and therefore I can't use other Web browser than one using Chromium engine.

Anyway, issue is closed.
